src/pages/LimitOrders/index.tsx

```diff
@@ -11,9 +16,15 @@ import {
 } from 'modules/limitOrders'
 import { OrdersTableWidget } from 'modules/ordersTable'
 import * as styledEl from 'modules/trade/pure/TradePageLayout'
+import { useWalletInfo } from 'modules/wallet'
+
+import { getIsNotComposableCowOrder } from 'utils/orderUtils/getIsNotComposableCowOrder'
 
 export default function LimitOrderPage() {
+  const { chainId, account } = useWalletInfo()
   const { isUnlocked } = useAtomValue(limitOrdersRawStateAtom)
+  const allLimitOrders = useOrders(chainId, account, OrderClass.LIMIT)
+  const onlyPlainLimitOrders = useMemo(() => allLimitOrders.filter(getIsNotComposableCowOrder), [allLimitOrders])
```

Comment:
Are composable cows created with class LIMIT or TWAP?

Reply:
Since `order-book` API returns all discrete orders, including TWAP parts we have to filter them.
The `order-book` API doesn't distinguish them, TWAP parts have `class=Limit`.
Because of it, we have to distinguish then on frontend side.

Reply:
Opened a thread here https://cowservices.slack.com/archives/C0375NV72SC/p1690199949260889